CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+
+**Attention:** All submissions must be in Markdown to be considered eligible for review.
+
+### Active voice
+Avoid passive voice; favor active voice. For example:
+
+    “Save your progress to prevent data loss.”
+
+        NOT
+
+    “Progress must be saved or data will be lost.”
+
+### Commands
+* Use code blocks for all input and output, and designate the correct programming language. When presenting commands in a sentence, use `monospace font`. 
+* When inserting code blocks in ordered lists, use the same indent as the copy in each step. For example:
+  
+  1. Install ModSec
+     ```shell
+     sudo yum install mod_security
+     ```
+  2. Restart Apache
+     ```shell
+     sudo service httpd restart
+     ```
+### Dates
+Spell out all dates. For example:
+
+    "November 15, 2018"
+     
+        NOT
+     
+    "11/15/18"
+ 
+### File and folders
+Use `monospace font` for all files, folders, paths, and directories.
+ 
+### Format
+See [Templates](#templates).
+ 
+### Headers
+Use headline-style capitalization for the title; use sentence-style for all others. Use H1 for the title, H2 for sections, H3 for sub-sections, and so on.
+
+### Images
+Any content relating to a UI other than the CLI requires images. If the UI-related content involves a procedure, use an image for each step. See [Using Images](#using-images) for more information.
+
+### Language
+As a general tip, please use direct, simple language. Omit needless words and phrases. 
+
+### Lists
+Use numbered lists for procedures. Break longer or complex procedures into sections with clear headers.
+
+### Naming conventions
+When creating the path for your article, do not include your metadata subject. For example, if you article was "Methods of Reindexing" Magento:
+
+    `docs/applications/methods-of-reindexing.md`
+
+        NOT
+        
+    `docs/applications/methods-of-reindexing-magento.md`
+    
+When creating the title of your article, provide more information. Continung the above example, the title would read as "Methods of Reindexing Magento" in H1 at the top of your article, after the metadata.
+
+### Point of view
+Use second-person language ("you"). Implied is favored in most cases. For example:
+
+    "Click **Save** to continue."
+    
+        NOT
+        
+    "You'll want to click **Save** to continue."
+    
+## Template
+See [Article Template](link)
+
+## Using Images
+Before uploading any images to the Thermo Repo:
+1. Crop the image appropriately. Balance size with the need to guide the reader.
+2. Make minimal but effective use of arrows, highlights, blurs, and other graphical tools to direct the reader. 
+
+To embed an image in your article, create a file in the images folder in your fork, then submit this file along with your article in the PR.
+
+1. In your fork, `<your_fork>/docs/images`, create a file that matches your article's name without the `.md` extension.
```

Comment:
"create a file" should be "create a directory"

Reply:
changed two occurrences of "create a file" to "create a directory."